From c7711f1b8f954f925ee00c4e30223547d547a84c Mon Sep 17 00:00:00 2001
From: Maximiliano Sandoval <msandova@protonmail.com>
Date: Sat, 12 Oct 2024 21:18:27 +0200
Subject: [PATCH] Remove _FOTIFY_SOURCE

Fixes:

python3 setup.py
CC: ['gcc'] (14, 2)
<command-line>: error: "_FORTIFY_SOURCE" redefined
<command-line>: note: this is the location of the previous definition
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e4c2496fd..d027b75e8 100755
--- a/setup.py
+++ b/setup.py
@@ -340,7 +340,7 @@ def init_env(
         'OVERRIDE_CFLAGS', (
             f'-Wextra {float_conversion} -Wno-missing-field-initializers -Wall -Wstrict-prototypes {std}'
             f' {werror} {optimize} {sanitize_flag} -fwrapv {stack_protector} {missing_braces}'
-            f' -pipe {march} -fvisibility=hidden {fortify_source}'
+            f' -pipe {march} -fvisibility=hidden'
         )
     )
     cflags = shlex.split(cflags_) + shlex.split(
-- 
2.47.0

